test/color/test_hls.py

```diff
@@ -91,9 +91,12 @@ def test_unit(self, device, dtype):
             dtype=dtype,
         )
 
-        assert_close(kornia.color.rgb_to_hls(data), expected)
+        self.assert_close(kornia.color.rgb_to_hls(data), expected)
 
     def test_nan_rgb_to_hls(self, device, dtype):
+        if dtype == torch.float16:
+            pytest.skip('not work for half-precision')
```

Comment:
@MrShevan can you open issues and tag as improvements (not bugs) with the all the functions not supported by half precision?

Reply:
https://github.com/kornia/kornia/issues/1830 Done it! But I can't add label `improvement`